src/main/java/seedu/address/logic/commands/deletestudentcommands/DeleteStudentByEmailCommand.java

```diff
@@ -61,4 +69,15 @@ public String toString() {
                 .add("email", email)
                 .toString();
     }
+
+    @Override
+    public void deleteStudentFromTutorialClasses(Model model, Person student) {
```

Comment:
Can this method be abstracted out to the parent class? Seems like it is duplicated code across all sub classes of DeleteStudentCommand.